state_management/csharp/sdk/order-processor/Program.cs

```diff
@@ -12,7 +12,7 @@
 
 string DAPR_STORE_NAME = "statestore";
 var client = new DaprClientBuilder().Build();
-for (int i = 1; i <= 100; i++) {
+for (int i = 1; i <= 10; i++) {

```

Comment:
I think we should leave 100.  Even though it's long and slows tests, this is used by Resiliency Quickstart so you have time to try failures and then restore.

Reply:
Fixed

Reply:
It is done, I can't find the evidence with 10 anywhere; The last GH action run also shows the no. of records as 100. 

![image](https://github.com/dapr/quickstarts/assets/46513504/57c705ec-554a-4668-901f-1f65902bddc8)